public/components/common/hocs/withUserLogged.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Wazuh app - React HOC to manage if the user is logged in
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+import React from "react";
+import { useSelector } from 'react-redux';
+import { EuiProgress, EuiText,EuiSpacer } from '@elastic/eui';
+import { getHttp } from '../../../kibana-services';
+
+
+
+export const withUserHasLogged = (WrappedComponent) => (props) => {
```

Comment:
Maybe the name of the const should be the same than the name or the file.

Reply:
It was fixed, thanks you!